core/src/main/java/org/apache/calcite/sql/SqlCreateOrReplace.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ *
+ */
+package org.apache.calcite.sql;
+
+/**
+ * Enum used by {@link org.apache.calcite.sql.SqlCreate} to indicate whether the query is a
+ * "CREATE", "CREATE OR REPLACE", or "REPLACE" query
+ */
+public enum SqlCreateOrReplace {
+  CREATE("CREATE"),
+  REPLACE("REPLACE"),
+  CREATE_OR_REPLACE("CREATE OR REPLACE");
+
+  private final String name;
```

Comment:
Why use strings at all? Can we just use the raw enum and get rid of the rest of these helper methods in the enum definition?

Reply:
My idea was to simplify the unparsing logic by making it possible to convert the enum directly to the relevant keywords (rather than creating a switch and writing unparsing code for each case).